src/lib/paginator.svelte

```diff
@@ -47,4 +60,5 @@
 			>Go &rarr;</button
 		>
 	</div>
+	<!-- <input type="number" bind:value={builder.paginator.limit}/> -->
```

Comment:
Is this comment needed, can this line just be removed?

Reply:
yes